### DISCUSSION TITLE
comment commits with output of a job

### DISCUSSION THREAD

**Discussion by rcastellotti** [community/community#25204] [Actions]
<p>Hi everybody,</p>
<p>Recently i wrote this action to lint some python code.</p>
<pre><code>name: kitto_linter

on:
  push:
    branches: [master]
  pull_request:
    branches: [master]

jobs:
  lint:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Set up Python 3.8
      uses: actions/setup-python@v1
      with:
        python-version: 3.8

    - name: checking requirements.txt is ok
      run: |
          python -m pip install --upgrade pip
          pip install -r requirements.txt
    - name: purring and buying corn-flakes
      run: pip install flake8

    - name: kitto is linting
      run: flake8 .
</code></pre>
<p> This just does the job, but I was wondering if I could implement something that comments the event that triggered the action (commit on master or pr) with the output of a command I run (specifically <em>flake8 . this time).</em></p>
<p>While looking around I found <a href="https://github.com/marketplace/actions/commit-comment" title="action" rel="nofollow noopener">this action  </a>however I can’t understand how to use a variable in the body ( i.e. I could assign output = flake8 . &gt; flake8_output.txt).</p>
<p>How can I do this? Am I missing something?</p>
<p>Thanks in advance.</p>

**Comment by jenschelkopf** [comment, marked as answer]
<p>The types of input that are supported by an action is up to the author.</p>
<p>It looks like “Commit Comment” supports plain text and environment variables, but doesn’t support reading from a file.</p>
<p>Please reach out to the author for additing support for files.</p>

**Comment by rcastellotti** [comment]
<p>Yea, you are right, probably it’s the best thing to do</p>
<p>Thanks!</p>
